src/core/qgsrelation.h

```diff
@@ -272,6 +280,15 @@ class CORE_EXPORT QgsRelation
      */
     bool isValid() const;
 
+    /**
+     * Compares the two QgsRelation, ignoring the name and the ID.
+     *
+     * @param other The other relation
+     * @return true if they are similar
+     * @note added in QGIS 3.0
+     */
+    bool isSame( const QgsRelation& other ) const;
```

Comment:
What's the problem with `isSame`? I've seen in other APIs use it and I don't see why the method should have a name that works for both english and french...